ebuild-writing/functions/src_test/text.xml

```diff
@@ -59,13 +59,13 @@ src_test() {
 <body>
 <codesample lang="ebuild">
 src_test() {
-    cd "${S}"/src/testdir
+    cd "${S}"/src/testdir || die
 
     # Test 49 won't work inside a portage environment
-    sed -i -e 's~test49.out~~g' Makefile
+    sed -i -e 's~test49.out~~g' Makefile || die
 
-    # Try to run the non-gui tests only
-    make test-nongui \
+    # Try to run the non-gui tests only, non-parallel
```

Comment:
Should we rather say `for non-parallelism`? I'm guessing this is related to the hunk about parallelism you deleted below.